devicemapper/thin_pool_watcher.go

```diff
@@ -163,14 +163,14 @@ func (w *ThinPoolWatcher) checkReservation(poolName string) (bool, error) {
 		return false, err
 	}
 
-	tokens := strings.Split(string(output), " ")
-	// Split returns the input as the last item in the result, adjust the
-	// number of tokens by one
-	if len(tokens) != thinPoolDmsetupStatusTokens+1 {
-		return false, fmt.Errorf("unexpected output of dmsetup status command; expected 11 fields, got %v; output: %v", len(tokens), string(output))
+	// we care about the field at fields[thinPoolDmsetupStatusHeldMetadataRoot],
+	// so make sure we get enough fields
+	fields := strings.SplitN(string(output), " ", thinPoolDmsetupStatusMinTokens+1)
```

Comment:
Why +1 here? Doesn't look like the last field is used? (`thinPoolDmsetupStatusMinTokens` is already incremented from Root)